moban/copy/__init__.py

```diff
@@ -30,7 +29,7 @@ def get_template(self, template_file):
         real_file_name = utils.find_file_in_template_dirs(
             template_file, self.template_dirs
         )
-        with codecs.open(real_file_name, encoding="utf-8") as file_handle:
+        with open(real_file_name, "rb") as file_handle:
             return file_handle.read()
```

Comment:
Referring https://github.com/moremoban/moban/issues/265#issuecomment-485063029:

> When saving, it should use 'wb' flag, treating the content as binary throughout the journey within moban.

We would write into the file here.